tests/functional/queries.js

```diff
@@ -176,7 +180,8 @@ const screenAuthenticationInApp = {
       new QmlQueryComposer('//authVerificationSessionByTotpNeeded-getHelpLink'),
   AUTH_TOTP_BACK_BUTTON:
       new QmlQueryComposer('//authVerificationSessionByTotpNeeded-backButton'),
-  AUTH_TOTP_PASTE_BUTTON: new QmlQueryComposer('//authVerificationSessionByTotpNeeded-inputPasteButton'),
+  AUTH_TOTP_PASTE_BUTTON: new QmlQueryComposer(
```

Comment:
Why this changes? We should use clang-format

Reply:
This is the result of running clang format on this file. I think it moved `//authVerificationSessionByTotpNeeded...` to a new line since it exceeds the line length